### PR TITLE
doc(README): better explain component grouping

### PR DESCRIPTION
**Pull request by corinnekrych**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,12 +101,21 @@ For example, while working on `new dashboard` feature, Adam wanted to share the
...
 ### Compoment
 `ngx-feature-flag` provides 2 components to help you hide and show your features under feature-flag.
+
+> <b>TIP: As a best practise,</b> if you know a page of your app will have several features flag, group them per page. We use a naming convention:
+`Page.MyFeature` where `Page` is the name you use in the FeatureFlagResolver. 
+
 #### use case 1: My feature is a component I want to hide/show
-For example, let's hide `EnvironmentWidgetComponent`.
-* In unleash admin ui, add your new feature with a `featureName` (here `Test`), associate a `enableByLevel` strategy, enter the `level` parameter (`internal`, `experimental`, `beta`, `released`). If this level (for ex: experimental) is below your user-consent level (for ex: beta), you won't see the component. 
+For example, let's hide `EnvironmentWidgetComponent` which is shown in a `Enviroment` page. Follow the best pratices and name your feature `Enviroment.Test`. If your don't want to "group" features, you can call it simply `Test`.
```
Comment (aptmac) [inline README.md]: Couple of typos in this line:

s/Enviroment/Environment
s/pratices/practices

**Comment by corinnekrych** [comment]
thanks @aptmac for the review. ok to merge?

File: README.md [hunk 2]
```diff
@@ -102,12 +101,21 @@ For example, while working on `new dashboard` feature, Adam wanted to share the
 
 ### Compoment
 `ngx-feature-flag` provides 2 components to help you hide and show your features under feature-flag.
+
+> <b>TIP: As a best practise,</b> if you know a page of your app will have several features flag, group them per page. We use a naming convention:
```
Comment (aptmac) [inline README.md]: Should "several features flag" be written "several feature flags" ?

**Comment by aptmac** [comment]
@corinnekrych I found 1 other grammatical error, I think it looks good to go after that!

**Comment by corinnekrych** [comment]
thanks! @aptmac

**Pull request merged**